### PR TITLE
README.md to describe RFCs at O(1) Labs

### PR DESCRIPTION
**Pull request by barriebyron**
This PR creates the first draft of our RFC repository welcome sign: the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+- RFCs happen downstream from product requirements doc (PRDs).
+
+## Workflow
+
+1. Product teams define product requirements doc in the product requirements doc (PRD).
+2. An internal O(1) Labs team member creates a pull request using the [0000-template.md](0001-template.md).
+3. The RFC covers the changes in enough detail to inform the decision.
+4. Discussions occur in the RFC pull request within a rapid defined time period.
+5. If architecture changes proposed in the initial RFC are significantly changed during the discussion, the submitter updates the RFC in a separate pull request.
+6. All architects must approve the RFC pull request within two business days.
```
Comment (bkase) [inline README.md:27]:
```suggestion
6. All architects must respond to the RFC pull request within two business days.
```

**Review by bkase — APPROVED** [review]
This is amazing!

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+- The rapid review and approval cycle prevents lengthy opportunities for public input. 
+- In contrast to timeless product specifications, RFCs are more ephemeral and benefit from the narrower focus of a delta.
+- In many software projects, a typical RFC invites public discourse. In contrast, an RFC at O(1) Labs is an internal design document to ensure that all architects participate in design decisions. 
+- RFCs happen downstream from product requirements doc (PRDs).
+
+## Workflow
+
+1. Product teams define product requirements doc in the product requirements doc (PRD).
+2. An internal O(1) Labs team member creates a pull request using the [0000-template.md](0001-template.md).
+3. The RFC covers the changes in enough detail to inform the decision.
```
Comment (nicc) [inline README.md:24]:
```suggestion
3. The RFC describes changes in enough detail to inform the decision. This includes links to proposed changes to persistent or cannonical design records such as specs and code comments.
```

**Review by nicc — COMMENTED** [review]
Looks great, @barriebyron! Have made a couple of suggestions.

**Comment by barriebyron** [comment]
related to https://github.com/o1-labs/rfcs/pull/3

**Pull request merged**